modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/pagemem/ExponentialBackoffMemoryProtectionThrottle.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021 GridGain Systems, Inc. and Contributors.
+ *
+ * Licensed under the GridGain Community Edition License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.gridgain.com/products/software/community-edition/gridgain-community-edition-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.internal.processors.cache.persistence.pagemem;
+
+/**
+ * Logic used to protect memory (mainly, Checkpoint Buffer) from exhaustion using exponential backoff.
+ */
+class ExponentialBackoffMemoryProtectionThrottle {
```

Comment:
Lets rename the class to `ExponentialBackoffThrottlingStrategy`.
Reasoning:
1. It is multipurpose and can be used not only to protect free memory.
2. We shouldn't use the same name (Throttle) for entities of two types: that encapsulate logic of park time calculation and that do actual throttling (PagesWriteThrottle, PagesWriteSpeedBasedThrottle).

Reply:
Renamed the class